src/adapters/concordance/adapter.js

```diff
@@ -0,0 +1,156 @@
+import DefaultConfig from '@/adapters/concordance/config.json'
+import AuthorWorkConfigConfig from '@/adapters/concordance/author-work.json'
+
+import { ResourceProvider, Author, WordUsageExample } from 'alpheios-data-models'
+import BaseAdapter from '@/adapters/base-adapter'
+
+class AlpheiosConcordanceAdapter extends BaseAdapter {
+  /**
+   * Adapter uploads config data and creates provider
+   * @param {Object} config - properties with higher priority
+  */
+  constructor (config = {}) {
+    super()
+    this.config = this.uploadConfig(config, DefaultConfig)
+    this.provider = new ResourceProvider(this.config.url, this.config.rights)
+    this.authors = []
+  }
+
+  /**
+  * This method retrieves a list of available authors and textWorks.
+  * For now it uploads data from json file, but later it will fetch data from cordance api
+  * @return {[Author]}
+  */
+  async getAuthorsWorks () {
+    try {
+      this.authorWorkData = await this.uploadConfig({}, AuthorWorkConfigConfig)
```

Comment:
@balmas I suggest, not to upload this in constructor, but only simply:
1) add additional argument `reload` - that could force reload data from source
2) add check (if not reload and authors are already filled) - load only one time

Because, by the sense of this method - we should upload data from some API (may be with some filters by first letters or by workTitle) about authors in future.
And this operation could be not so lightweight. 

And also there could be a way to use this feature without any uploading author data.
For example, we will decide to show only cit for wordUsageExample for mobile, as uploading and storing the whole data about authors is not good for mobile RAM.

I will make some little fixes to getWordUsageExamples method to be able to use without authors data.
What do you think?

Reply:
ok, that makes sense.